<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PBCBoxObjectDesigner</class>
 <widget class="QWidget" name="PBCBoxObjectDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PBCBox</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="_4">
       <item>
        <widget class="QGroupBox" name="generalGB">
         <property name="title">
          <string>General</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="flagsGB">
         <property name="title">
          <string>Flags</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="visibleCB">
            <property name="text">
             <string>Visible</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QCheckBox" name="showFamiliarCB">
            <property name="text">
             <string>Familiar shape</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="showCenterCB">
            <property name="text">
             <string>Show box center</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QCheckBox" name="centerToOriginCB">
            <property name="text">
             <string>Center to origin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="selectedCB">
            <property name="text">
             <string>Highlighted</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="positionGB">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QGridLayout" name="_3">
          <item row="0" column="1">
           <widget class="QSpinBox" name="kaSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="kbSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QSpinBox" name="kcSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>KA:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>KB:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>KC:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="objectsTab">
      <attribute name="title">
       <string>Objects</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="setupTab">
      <attribute name="title">
       <string>Setup</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="refByTab">
      <attribute name="title">
       <string>Reference by</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
